src/nfa/operations.cc

```diff
@@ -671,52 +657,52 @@ Nfa Mata::Nfa::Algorithms::minimize_brzozowski(const Nfa& aut) {
 
 Nfa Mata::Nfa::minimize(
                 const Nfa& aut,
-                const StringMap& params)
+                const ParameterMap& params)
 {
-	Nfa result;
-	// setting the default algorithm
-	decltype(Algorithms::minimize_brzozowski)* algo = Algorithms::minimize_brzozowski;
-	if (!haskey(params, "algorithm")) {
-		throw std::runtime_error(std::to_string(__func__) +
-			" requires setting the \"algo\" key in the \"params\" argument; "
-			"received: " + std::to_string(params));
-	}
-
-	const std::string& str_algo = params.at("algorithm");
-	if ("brzozowski" == str_algo) {  /* default */ }
-	else {
-		throw std::runtime_error(std::to_string(__func__) +
-			" received an unknown value of the \"algo\" key: " + str_algo);
-	}
-
-	return algo(aut);
+    Nfa result;
+    // setting the default algorithm
+    decltype(Algorithms::minimize_brzozowski)* algo = Algorithms::minimize_brzozowski;
+    if (!haskey(params, "algorithm")) {
+        throw std::runtime_error(std::to_string(__func__) +
+            " requires setting the \"algo\" key in the \"params\" argument; "
+            "received: " + std::to_string(params));
+    }
+
+    const std::string& str_algo = params.at("algorithm");
+    if ("brzozowski" == str_algo) {  /* default */ }
+    else {
+        throw std::runtime_error(std::to_string(__func__) +
+            " received an unknown value of the \"algo\" key: " + str_algo);
+    }
+
+    return algo(aut);
 }
 
 Nfa Mata::Nfa::uni(const Nfa &lhs, const Nfa &rhs) {
     Nfa unionAutomaton = rhs;
 
-    StateToStateMap thisStateToUnionState;
+    StateRenaming this_state_to_union_state;
```

Comment:
lhs_state_renaiming, rhs_state_renaming?

Reply:
I will rename the old Vata names for variables used in this function in this PR, then.